docs/v0.5.x/plugin-packs.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Plugin Packs
+---
+
+### What is a Plugin Pack?
+
+An ember-cli-deploy plugin pack is an ember-cli addon that will auto install a collection of ember-cli-deploy plugins to satisfy a particular deployment strategy, so you don't have to.
+
+### Why use a Plugin Pack?
+
+There are many different [deployment strategies](../deployment-strategies-overview) and setups out there, however we are seeing some common strategies emerging. Plugin packs are designed to encapsulate the logic of these
+deployment strategies and give you all the plugins you need to benefit from a particular deployment strategy.
+
+A great example of a common deployment strategy is, what has in recent times been coined, [The Lightning Approach](https://www.youtube.com/watch?v=QZVYP3cPcWQ) to deployment. This strategy involves pushing an
+index.html file to Redis and assets to S3. But there are also a bunch of other things that you would want to do with this approach. Therefore, you probably would want to install all
+of the following plugins:
+
+- [ember-cli-deploy-build](https://github.com/ember-cli-deploy/ember-cli-deploy-build) - to build your app
+- [ember-cli-deploy-revision-data](https://github.com/ember-cli-deploy/ember-cli-deploy-revision-data) - to generate a revision key
+- [ember-cli-deploy-gzip](https://github.com/ember-cli-deploy/ember-cli-deploy-gzip) - to gzip files
+- [ember-cli-deploy-manifest](https://github.com/ember-cli-deploy/ember-cli-deploy-manifest) - to generate a manifest
+- [ember-cli-deploy-redis](https://github.com/ember-cli-deploy/ember-cli-deploy-redis) - to upload to redis
+- [ember-cli-deploy-s3](https://github.com/ember-cli-deploy/ember-cli-deploy-s3) - to upload to s3
+- [ember-cli-deploy-display-revisions](https://github.com/ember-cli-deploy/ember-cli-deploy-display-revisions) - to display a list of deployed revisions
+
+To make it easier to get up and running with this deployment strategy, you could instead just install the [ember-cli-deploy-lightning-pack](https://github.com/ember-cli-deploy/ember-cli-deploy-lightning-pack). This pack will install the above plugins automatically
+and provide you with a blueprint `config/deploy.js`.
+
+### How do I use a Plugin Pack?
+
+A plugin pack is just an ember-cli addon so you just need to install it like you would any addon:
+
+```bash
+ember install ember-cli-deploy-lightning-pack
+```
+
+From then on, just carry on deploying with [The Lightning Strategy](../the-lightning-strategy).
+
+### Why would I create a Plugin Pack?
+
+There are a couple of reason why you might want to create your own plugin pack.
+
+The first might be that you want to package up your deployment strategy so it's easy for you to implement across multiple ember-cli projects. This would save you the time of having to install all of the
+required plugins in every project.
```

Comment:
Add "We envision most companies that support multiple Ember apps creating their own plugin pack for internal use."